config/features.yml

```diff
@@ -244,26 +244,18 @@ features:
     actor_type: user
     description: >
       This determines when the wizard should show up on the form 526 intro page
-  show_new_get_medical_records_page:
+  cerner_override_668:
     actor_type: user
     description: >
-      This will show the non-Cerner-user and Cerner-user content for the page /health-care/get-medical-records/
-  show_new_refill_track_prescriptions_page:
+      This will show the Cerner facility 668 as `isCerner`.
+  cerner_override_757:
     actor_type: user
     description: >
-      This will show the non-Cerner-user and Cerner-user content for the page /health-care/refill-track-prescriptions/
+      This will show the Cerner facility 757 as `isCerner`.
   show_new_schedule_view_appointments_page:
     actor_type: user
     description: >
       This will show the non-Cerner-user and Cerner-user content for the page /health-care/schedule-view-va-appointments/
-  show_new_secure_messaging_page:
```

Comment:
I removed the following feature toggles since they are not used:

```
show_new_get_medical_records_page
show_new_refill_track_prescriptions_page
show_new_secure_messaging_page
show_new_view_test_lab_results_page
```

We currently still use `show_new_schedule_view_appointments_page`, but I'll send up a follow-up PR post-Cerner launch to remove that too.

I added the following feature toggles:

```
cerner_override_668
cerner_override_757
```

These are used to force `isCerner` to true for these 2 facilities.